images/win/scripts/Tests/Rust.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+Describe "Rust" {
+    Push-Location -Path $env:UserProfile
+    if ( -not ((Test-Path -Path ".rustup") -and (Test-Path -Path ".cargo")))
+    {
+        $null = New-Item -Name ".rustup" -Value "C:\Rust\.rustup" -ItemType Junction
+        $null = New-Item -Name ".cargo" -Value "C:\Rust\.cargo" -ItemType Junction
+    }
+    Pop-Location

```

Comment:
I don't think that we should change the image in tests. Can we avoid it somehow?

Reply:
Without temporary `Junction` point we can't execute tools from Rust. I fixed Rust test to remove invoke the commands and to stay only validate binary path.